### PR TITLE
Leap 15.x New splash animation

### PR DESCRIPTION
**Pull request by hellcp**
Splash animation following guidelines with flickers and changing background.

### CODE REVIEW

**Comment by Vogtinator** [comment]
You can remove the `*.svgz` files, they are part of the upstream branding so will already be available.

File: config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml [hunk 1]
```diff
@@ -95,24 +95,33 @@ Rectangle {
...
+            property: "color"
+            from: "black"
+            to: "#081c2d"
+            duration: 1000
+            easing.type: Easing.InExpo
+        }
+        OpacityAnimator {
+            id: castingIntroAnimation
+            running: false
+            target: casting
```
Comment (Vogtinator) [inline config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml]: Typo: `ReferenceError: casting is not defined`.

**Review by Vogtinator — CHANGES_REQUESTED** [review]

**Comment by Vogtinator** [comment]
I gave it a try, it works now.

Technical issues:
- The Bulb-License file needs to be updated

My opinion:
- The timing is off. You don't even see the blur effect and the flickering only last for approx. 0.05 s...
- The bulb doesn't glow, it just looks weird

**Comment by Vogtinator** [comment]
I still don't like the speed - it looks like an Adam7 PNG loading over a very slow connection in the beginning.

**Comment by Vogtinator** [comment]
Heh, I meant that the animation is over too quickly, it needs to be at least three times as long IMO.

**Comment by Vogtinator** [comment]
Looks better, but I much prefer glowing over the blur. A bulb doesn't usually blur when it's turned off in real life ;-)

**Comment by hellcp** [comment]
I like blur, it's much closer to what I wanted to do with it ;)

**Comment by lbeltrame** [comment]
As I said on IRC, perhaps the "light up" is a little bland. I don't have any other objections.

File: config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml [hunk 2]
```diff
@@ -82,7 +115,7 @@ Rectangle {
         Image {
             id: busyIndicator
             anchors.horizontalCenter: parent.horizontalCenter
-            y: parent.height * 0.7
+            y: parent.height * 0.75
             source: "images/busywidget.svgz"
```
Comment (antlarr) [inline config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml:96]: I think this file was forgotten. Shouldn't it be added in the commit, too?

Comment (hellcp) [reply config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml:96]: It's per design, it should use files from upstream theme and replace just provided ones

Comment (antlarr) [reply config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml:96]: ah ok, I moved away the directory when testing your changes and I should have just copied it over. It's ok then.

**Comment by Vogtinator** [comment]
The distributor logo has a white area instead of being transparent - is that intentional?
There's also a darker line at the bottom.

**Comment by Vogtinator** [comment]
IMO the flicker at the beginning is now too slow - it doesn't look like a flicker but more like a graphical glitch or "retry".

**Comment by Vogtinator** [comment]
Here's how I'd prefer it:

```
diff --git a/config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml b/config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml
index 40ceddf..16ba667 100644
--- a/config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml
+++ b/config-files/usr/share/plasma/look-and-feel/org.openSUSE.desktop/contents/splash/Splash.qml
@@ -34,8 +34,6 @@ Rectangle {
         } else if (stage == 3) {
             casingIntroAnimation.running = true;
             highlightIntroAnimation.running = true;
-        } else if (stage == 4) {
-            highlightIntroReanimation.running = true;
         } else if (stage == 5) {
             introAnimation.target = busyIndicator;
             introAnimation.from = 1;
@@ -70,7 +68,7 @@ Rectangle {
     }
     Image {
         id: casing
-        opacity: 0.05
+        opacity: 0.1
         source: "images/bulb.svg"
         fillMode: Image.PreserveAspectFit
         height: parent.height * 0.5
@@ -133,7 +131,7 @@ Rectangle {
             target: casingBlur
             from: 0
             to: 1
-            duration: 1500
+            duration: 800
             easing.type: Easing.InBounce
         }
         OpacityAnimator {
@@ -141,18 +139,8 @@ Rectangle {
             running: false
             target: highlightBlur
             from: 0
-            to: 0.5
-            duration: 2000
-            easing.type: Easing.InBounce
-        }
-        OpacityAnimator {
-            id: highlightIntroReanimation
-            running: false
-            target: highlightBlur
-            from: 0.2
             to: 1
-            duration: 2000
-            easing.type: Easing.InExpo
+            duration: 1200
+            easing.type: Easing.InBounce
         }
-        
 }

```

**Review by Vogtinator — APPROVED** [review]

**Comment by Vogtinator** [comment]
> The distributor logo has a white area instead of being transparent - is that intentional?

Didn't get fixed - can you do that?

**Comment by hellcp** [comment]
it is intentional, the same way how I did it in case of Gnome Boxes:
Screenshot

**Comment by Vogtinator** [comment]
I suppose it might make sense for dark themes, but on bright themes it looks out of place IMO.

If you change it, how does it look for dark themes?

**Comment by hellcp** [comment]
Created mockup (because it's quicker than installing plasma honestly) with both adwaita flavors
lightanddarkthemes
Looks fine to me :D

**Comment by Vogtinator** [comment]
Can you do the same with the white background removed for comparison?

**Comment by hellcp** [comment]
lightanddark

**Comment by tigerfoot** [comment]
Transparency is the norm.
so no white please.

**Comment by javierllorente** [comment]
+1

**Comment by Vogtinator** [comment]
The majority seems to be for the transparent icon, can you change it?

**Pull request merged**